docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -149,9 +152,77 @@ an expert; this is meant to be accessible for all people. Wherever you are in th
 journey, this guide is intended to be a clear, helpful resource.  
 
 We’re not going to put any words into other people’s mouths, but we will use our own voice to do 
-the work and use the terms we find most accurate and inclusive.  
+the work and use the terms we find most accurate and inclusive.
+
+1.2 Introduction to sex and gender
+----------------------------------
 
-1.2 Glossary of General Terminology
+Most human fetuses have either two X chromosomes or one X and one Y chromosome.
+A fetus' configuration of X and Y chromosomes influences a number of aspects of development,
+such as hormones, gonads, and genitalia.
+Then the child is born into a society with lots of pre-existing ideas about how bodies should
+look and work, how people should behave, and how all these things should relate to one another.
+Almost immediately, expectations and norms about identity and behavior are placed on the child,
```

Comment:
Some are even in place before birth, based on genetic testing!  No need to make this even more complicated, though.

Reply:
It's funny that you pointed this out because in an earlier version I had noted this.

There is even research showing that [parents begin treating their children differently in the womb](https://www.jstor.org/stable/23799108). But as Abie said I was trying to streamline this paragraph.